presto-tests/src/test/java/com/facebook/presto/tests/TestNoisyAggregations.java

```diff
@@ -27,6 +32,13 @@ protected QueryRunner createQueryRunner()
         return TpchQueryRunnerBuilder.builder().build();
     }
 
+    @Override
+    protected QueryRunner createExpectedQueryRunner()
```

Comment:
Why is this needed?

Reply:
Thanks. Short answer is to make sure that the actual and expected queries are run with the same runner.

queryRunner is a TpchQueryRunnerBuilder (which I assume would give us TpchQueryRunner)
expectedQueryRunner is by default a H2QueryRunner. 

I am not super what their difference is, but I think it is better for them to be the same.

In addition, in my WIP code to implement the next function in this series, noisy_avg_gaussian, **without** this override to make sure they are the same runner, I got an error where:
- `SELECT noisy_avg_gaussian(linenumber, 0) FROM lineitem` returns 3.004270876609888
- `SELECT avg(linenumber) FROM lineitem` returns 3.0

I needed this override to get that test working. So I think it is better for them to be the same anyway.

Reply:
Discussed offline, let's address this in the PR for noisy_avg_gaussian. H2QueryRunner is returning 3.0 for `SELECT avg(linenumber) FROM lineitem` which is weird and needs to be investigated. However, we can do that in parallel and not block this PR

Reply:
3.0 is fishy, maybe its a type issue? can you try avg(linenumber+.0)?

Reply:
Looks like a type issue, avg(linenumber+.0) returns the correct value. So its ok to use TpchQueryRunner to compare restults, and in the comment we can add that H2QueryRunner is doing averages as ints instead of floats(not sure why).

Reply:
yeah, I tried avg(linenumber + 0.0) or avg(cast(linenumber as double)) and both worked. 

Just for the next PR for avg, do you suggest overriding expectedQueryRunner or casting to double like this?

Reply:
@duykienvp overriding is fine.